src/textual/widgets/_button.py

```diff
@@ -239,22 +240,29 @@ async def _on_click(self, event: events.Click) -> None:
         event.stop()
         self.press()
 
-    def press(self) -> Self:
+    def press(self, *, level: AnimationLevel = "basic") -> Self:
```

Comment:
After a discussion in the office, we don't think this qualifies as an animation at all

Reply:
👌